### PR TITLE
Comment and logging change only

### PR DESCRIPTION
**Pull request by mogren**
*Description of changes:*
* Use ENI and IP consistently instead of mixing cases.
* Some minor spelling fixes as well.
* Removed some excessive newlines

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: pkg/networkutils/network.go [hunk 1]
```diff
@@ -622,7 +618,7 @@ func setupENINetwork(eniIP string, eniMAC string, eniTable int, eniSubnetCIDR st
 						log.Errorf("Failed to add route %s/0 via %s table %d",
 							r.Dst.IP.String(), gw.String(), eniTable)
 						return errors.Wrapf(err,
-							"eni network setup: failed unable to add route %s/0 via %s table %d",
+							"ENI network setup: failed unable to add route %s/0 via %s table %d",
```
Comment (tiffanyfay) [inline pkg/networkutils/network.go]: This or a different PR (also to match the log.Errorf)
s/failed unable to add route/failed to add route/

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -152,7 +152,7 @@ func RetryNWithBackoff(backoff Backoff, n int, fn func() error) error {
 // If the number of tries is exhausted, the last error will be returned.
 func RetryNWithBackoffCtx(ctx context.Context, backoff Backoff, n int, fn func() error) error {
 	var err error
-	RetryWithBackoffCtx(ctx, backoff, func() error {
+	_ = RetryWithBackoffCtx(ctx, backoff, func() error {
```
Comment (tiffanyfay) [inline pkg/utils/utils.go]: What about erroring if it fails to retry?

File: pkg/networkutils/network.go (hunk 1 above)
Comment (mogren) [reply pkg/networkutils/network.go]: Thanks, good catch.

File: pkg/utils/utils.go (hunk 2 above)
Comment (mogren) [reply pkg/utils/utils.go]: I'll revert this. I think one issue is that we don't even use this code except in the tests. I'd much rather import the code from the official AWS go SDK once we actually use a retryer:

https://github.com/aws/aws-sdk-go/blob/master/aws/client/default_retryer.go

**Pull request merged**